### PR TITLE
[feature/RPA-938] Add metadata on LP

### PR DESCRIPTION
**Pull request by marcin-depcik**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by preludium — APPROVED** [review]

File: runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx [hunk 1]
```diff
@@ -51,9 +52,16 @@ export const getServerSideProps: GetServerSideProps<Props, Params> = async ({
         };
     }
 
+    const metadata: MetadataTags = {
+        title: `${post.title} | Runbotics Blog`,
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx:57]: have you checked if it's working for different languages?

File: runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx [hunk 2]
```diff
@@ -51,9 +52,16 @@ export const getServerSideProps: GetServerSideProps<Props, Params> = async ({
...
+        description: `RunBotics - ${post.title}`,
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx]: shouldn't it be post description? dobling titles don't make sense to me

File: runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx [hunk 3]
```diff
@@ -51,9 +52,16 @@ export const getServerSideProps: GetServerSideProps<Props, Params> = async ({
...
+        image: post.featuredImage?.url,
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx]: how about backup universal RB image? 
image: post.featuredImage?.url ?? backupRunBoticsUrl,

File: runbotics/runbotics-orchestrator-ui/src/pages/index.tsx [hunk 4]
```diff
@@ -26,9 +27,14 @@ export const getServerSideProps: GetServerSideProps<Props> = async ({ res, local
 
     const blogPosts = cache.posts?.slice(0, 3) ?? [];
 
+    const metadata: MetadataTags = {
+        image: HeroBg.src,
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/pages/index.tsx:32]: doesn't it missing the title?

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/Matadata/Metadata.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+interface Metadata {
+    metadata: MetadataTags;
+}
+
+const DEFAULT_TAG_CONTENT = 'RunBotics';
+
+const Metadata: FC<Metadata> = ({ metadata }) => {
+    const getContent = (tag: string) =>
+        metadata ? (metadata[tag] ?? DEFAULT_TAG_CONTENT) : DEFAULT_TAG_CONTENT;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/Matadata/Metadata.tsx]: imo 
(metadata && metadata[tag]) ?? DEFAULT_TAG_CONTENT 
is simpler

**Review by star-in-the-jar — CHANGES_REQUESTED** [review]
bump version

File: runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx (hunk 1 above)
Comment (marcin-depcik) [reply runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx:57]: yes, it works, both for en and pl translation

File: runbotics/runbotics-orchestrator-ui/src/pages/index.tsx (hunk 4 above)
Comment (marcin-depcik) [reply runbotics/runbotics-orchestrator-ui/src/pages/index.tsx:32]: no, home page gets default metadata from Metadata component, but I added custom description content

**Review by star-in-the-jar — APPROVED** [review]

**Pull request merged**